src/fsharp/NicePrint.fs

```diff
@@ -1108,13 +1108,20 @@ module private PrintTypes =
 
 /// Printing TAST objects
 module private PrintTastMemberOrVals = 
-    open PrintTypes
-    let private prettyLayoutOfMember denv typarInst (v:Val) = 
+    open PrintTypes 
+    let private prettyLayoutOfMemberShortOption denv typarInst (v:Val) short = 
```

Comment:
What is `short` referring to? `shortName`?

Reply:
Yes, it discards any additional information like "type" or "member"